src/App.tsx

```diff
@@ -1,26 +1,55 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { todosSlice } from './features/todos';
```

Comment:
Ensure that the `storeTodos` reducer in `todosSlice` updates the state immutably. Currently, it directly returns a new array, which might not be the best practice for immutability. Consider using the spread operator to merge the existing state with the new payload.